openfisca_france_local/parameters/departements/haute_saone/aide_permis.yaml

```
@@ -0,0 +1,52 @@
+age:
+  maximum:
```

Suite aux discussion sur une autre pr, je propose d'utiliser une terminologie qui ne perturbera pas trop les habitudes mais sere moins ambigue:
```suggestion
  maximum_inclusif:
```